google/cloud/aiplatform/training_jobs.py

```diff
@@ -4334,6 +4342,23 @@ def _model_upload_fail_string(self) -> str:
             "Model."
         )
 
+    @property
+    def evaluated_data_items_bigquery_uri(self) -> Optional[str]:
+        """BigQuery location of exported evaluated examples from the Training Job
+        Returns:
+            str: BigQuery uri for the exported evaluated examples if the export
+                feature is enabled for training.
+            None: If the export feature was not enabled for training.
+        """
+
+        self._assert_gca_resource_is_available()
+
+        metadata = self._gca_resource.training_task_metadata
+        if "evaluatedDataItemsBigqueryUri" in metadata:
```

Comment:
Safer to do this: `if metadata and "evaluatedDataItemsBigqueryUri" in metadata:`

See this issue [b/192601601](b/192601601). `training_task_metadata` may not be populated by the service.

Reply:
Done.